lib/checks.js

```diff
@@ -23,37 +25,37 @@ function failStateCheck(callee, message, formatArgs) {
     failCheck(errors.IllegalStateError, callee, message, formatArgs);
 }
 
-module.exports.checkArgument = function(value, message) {
+module.exports.checkArgument = function checkArgument(value, message) {
```

Comment:
I don't like the stuttering this is introducing for sure. Maybe export all the functions together at the bottom of the file.